ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-stable-4.16.yaml

```diff
@@ -151,6 +151,21 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-proxy-cco-manual-security-token-service
+- as: aws-ipi-public-ipv4-pool-arm-f28
+  cron: 14 12 19 * *
+  steps:
+    cluster_profile: aws
```

Comment:
Is profile aws expected, or we'd like to use aws-qe ?

Reply:
Yeah, I think it is expected, because this specific install workflow need some special configurations on aws account level, for now is only the dev's account have the capacity.

Reply:
Ack